doc/README.md

```diff
@@ -11,10 +11,16 @@ The following are some helpful notes on how to run Bitcoin on your native platfo
 
 ### Unix
 
-You need the Qt4 run-time libraries to run Bitcoin-Qt. On Debian or Ubuntu:
+You need the Qt run-time libraries to run Bitcoin-Qt.
```

Comment:
Interesting. Good catch - this hasn't been true for a while. We now link qt statically on Linux, so there is no actual need to install any run-time library. 
Building is explained in `build-unix.md` so this part can go entirely.